src/tokenizer.rs

```diff
@@ -21,6 +21,7 @@ pub enum Symbol {
     Slash,
     Caret,
     Hash,
+    Concat,
```

Comment:
In the tokenizer, I've been trying to keep the names of the symbols separate from their actual semantics in Lua.

Is there a better name for `..` that doesn't describe what it does?

Reply:
Ergh, not sure. I noticed that, but couldn't think of another name. Possibly `TwoDots` or `TwoPeriods`?

Reply:
`DotDot`, vs `DotDotDot`? 😅 

`TwoDots` probably works.

Reply:
Alright; I've made the change!